client/devdocs/controller.js

```diff
@@ -1,43 +1,45 @@
 /**
  * External dependencies
  */
-var ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	qs = require( 'qs' ),
-	debounce = require( 'lodash/debounce' ),
-	page = require( 'page' ),
-	ReduxProvider = require( 'react-redux' ).Provider,
-	setSection = require( 'state/ui/actions' ).setSection,
-	EmptyContent = require( 'components/empty-content' );
+import ReactDom from 'react-dom';
+import React from 'react';
+import qs from 'qs';
+import debounce from 'lodash/debounce';
+import page from 'page';
+import { Provider } from 'react-redux';
```

Comment:
Was there a reason you chose to revert to naming this `Provider` ? I don't feel strongly either way, but had originally worried that `Provider` was less easy to understand at a glance than `ReduxProvider`.

As an FYI, you can rename ES6 imports using `as`:

```js
import { Provider as ReduxProvider } from 'react-redux';
```